src/environments/environment.ts

```diff
@@ -33,4 +33,5 @@ export const environment = {
   /** The following settings can be overridden by the `config.json` if present, see {@link AppSettings} */
   demo_mode: true,
   session_type: SessionType.mock,
+  authenticator: "keycloak", // set to "couchdb" if auth should happen with CouchDB directly
```

Comment:
should we make the default the other way round -> `CouchDB` on default and `Keycloak` has to be activated? This would make the migration with `config.json` caching issues easier.

Reply:
yes, let's do that :+1: 
Also in general I think it's nice to have the most lightweight setup as default.